src/tests/test_cubicspline.cc

```diff
@@ -72,6 +72,32 @@ BOOST_AUTO_TEST_CASE(cubicspline_fit_test) {
   
 }
 
+BOOST_AUTO_TEST_CASE(cubicspline_matrix_test) {
+    
+  CubicSpline cspline;   
+  cspline.setBCInt(0);
+  cspline.GenerateGrid(0.4,0.6,0.1);
+  Eigen::MatrixXd A=Eigen::VectorXd::Zero(1,6);
```

Comment:
this should be matrices and not vectors

Reply:
Sure!

Reply:
I also found and corrected another error in test_cubicspline.cc. Now it should work